guacamole-common-js/src/main/webapp/modules/SessionRecording.js

```diff
@@ -32,8 +32,13 @@ var Guacamole = Guacamole || {};
  * @param {!Blob|Guacamole.Tunnel} source
  *     The Blob from which the instructions of the recording should
  *     be read.
+ * @param {number} refreshInterval
+ *     The minimum number of milliseconds between updates to the recording
+ *     position through the provided onseek() callback. If non-positive, this
+ *     parameter will be ignored, and the recording position will only be
+ *     updated when seek requests are made, or when new frames are rendered.
```

Comment:
What are your thoughts on defaulting `refreshInterval` to 1000?

Reply:
Sure, fine by me.